packages/node-cli/messages.json

```diff
@@ -122,6 +126,33 @@
 	"COMMAND_LOCAL_RESOURCE_NOT_FOUND": "The process failed. The {0} file was not found in the project.",
 	"COMMAND_LOCAL_INVALID_XML": "Invalid xml {0}",
 
+	
+	"COMMAND_MANAGE_ACCOUNT_ERROR_CREDENTIALS_EMPTY": "There is no authentication ID (authID) configured for this account-role combination. Run \"suitecloud account:setup\" to set up a NetSuite account to use with SuiteCloud CLI for Node.js.",
```

Comment:
The first sentence of the message doesn't seem to be right, specially the account-role combination part.

Reply:
Hey David... The message was defined as this in the story, I have just copied it. Anyways I will check with Alba if it's the definitive.

Reply:
At the end... the message it's right